hpack/hpack.py

```diff
@@ -22,6 +22,7 @@
 INDEX_NONE = b'\x00'
 INDEX_NEVER = b'\x10'
 INDEX_INCREMENTAL = b'\x40'
+_PREFIX_BIT_MAX_NUMBERS = [(2 ** i) - 1 for i in range(10)]
```

Comment:
I'd like a comment here that acknowledges that there are too many entries in this list but that that's deliberate, and why it was done. =)